dirsrvtests/tests/suites/logging/logging_compression_test.py

```diff
@@ -0,0 +1,119 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2022 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+
+import glob
+import ldap
+import logging
+import pytest
+import os
+import time
+from lib389._constants import DEFAULT_SUFFIX
+from lib389.dseldif import DSEldif
+from lib389.topologies import topology_st as topo
+from lib389.idm.domain import Domain
+
+log = logging.getLogger(__name__)
+
+
+def log_rotated_count(log_type, log_dir, check_compressed=False):
+    # Check if the log was rotated
+    log_file = f'{log_dir}/{log_type}.2*'
+    if check_compressed:
+        log_file += ".gz"
+    return len(glob.glob(log_file))
+
+
+def update_and_sleep(suffix, sleep=True):
+    for loop in range(2):
+        for count in range(10):
+            suffix.replace('description', str(count))
+            suffix.get_attr_val('description')
+            with pytest.raises(ldap.OBJECT_CLASS_VIOLATION):
+                suffix.add('doesNotExist', 'error')
+        if sleep:
+            time.sleep(61)
```

Comment:
Could you please put a comment here on why we sleep 61 seconds?
While reviewing PR I haven't noticed the logrotationtime is set to 1 minute and I got confused for a bit.

Reply:
> Could you please put a comment here on why we sleep 61 seconds? While reviewing PR I haven't noticed the logrotationtime is set to 1 minute and I got confused for a bit.

Fixed